### PR TITLE
refactor: update for zeno-hub

### PR DESCRIPTION
**Pull request by cabreraalex**
# Description

Major refactor focused on zeno-hub.

fix ZEN-358

### CODE REVIEW

**Comment by Sparkier** [comment]
I removed the checks since the old ones mostly don't make sense now that we're moving to notebooks.

**Comment by Sparkier** [comment]
@neubig can you review this? I don't think anyone else can.

File: README.md [hunk 1]
```diff
@@ -1,104 +1,33 @@
...
-  A report comparing different methods
-  for creating chatbots, including API-based models such as ChatGPT and Cohere,
-  with open-source models such as Vicuna, Alpaca, and MPT.
-- **Translation** ([Report](examples/analysis_gpt_mt/report/),
-  [Browser](https://zeno-ml-translation-report.hf.space/)):
-  A report comparing GPT-based methods, Microsoft Translator, and the best system
-  from the Conference on Machine Translation.
+Each of the examples in this repository is specifically designed to be
+self-contained and easy to modify. To get started with
+zeno, we suggest that you find the closest example to what you're trying
```
Comment (neubig) [inline README.md:22]:
```suggestion
Zeno, we suggest that you find the closest example to what you're trying
```

**Review by neubig — APPROVED** [review]
LGTM with one minor typo fix!

**Pull request merged**